cadc-download-manager-server/build.gradle

```diff
@@ -23,17 +23,19 @@ dependencies {
     compile 'log4j:log4j:[1.2,)'
     compile 'javax.servlet:javax.servlet-api:[3.1,)'
 
-    compile 'org.opencadc:cadc-util:[1.0,)'
+    compile 'org.opencadc:cadc-util:[1.4.2,)'
     compile 'org.opencadc:cadc-log:[1.0,)'
     compile 'org.opencadc:cadc-registry:[1.0,)'
     compile 'org.opencadc:cadc-rest:[1.3.3,)'
     compile 'org.opencadc:cadc-app-kit:[1.0,)'
-    compile 'org.opencadc:cadc-download-manager:[1.4.0,)'
-    compile 'org.opencadc:cadc-web-util:[1.2.7,)'
+    compile 'org.opencadc:cadc-download-manager:[1.4.2,)'
+    compile 'org.opencadc:cadc-web-util:[1.2.10,)'
     compile 'commons-io:commons-io:[2.7,)'
 
     testCompile 'junit:junit:4.13'
     testCompile 'org.easymock:easymock:3.6'
+
+    runtime 'org.opencadc:cadc-access-control-identity:[1.1.0,)'
```

Comment:
this runtime dep should not be in the lib

Reply:
removed. dependency is in wwebui/downloadManager cadc9742 branch build.gradle file